README.md

```diff
@@ -1,6 +1,201 @@
 Job Restrictions Plugin for [Jenkins CI][1]
 =======================
-See [Job Restrictions Plugin Page][2] for more information.
+
+| Plugin Information |
+|---------------------------------------------------------------------------------------------------------------|
+| View Job Restrictions [on the plugin site](https://plugins.jenkins.io/job-restrictions) for more information. |
+
+**This plugin is up for adoption.** Want to help improve this plugin?
+[Click here to learn
+more](https://wiki.jenkins.io/display/JENKINS/Adopt+a+Plugin "Adopt a Plugin")!
```

Comment:
```suggestion
```

This is handled by github labels now, so shouldn't be in the readme.

Reply:
Thanks for the heads-up. I removed this section!